source/lib-intro.tex

```diff
@@ -3039,12 +3039,18 @@
 #define MB_CUR_MAX @\seebelow@
 
 namespace std {
+  // Exposition-only function type aliases
+  extern "C" using c_exit_t = void();                                // \expos
+  extern "C++" using cxx_exit_t = void();                            // \expos
```

Comment:
This is the only use of "cxx" to refer to C++ in the standard; we use "cpp" in the few places where this comes up. Perhaps we could just omit the `cxx_` prefix and maybe also the `extern "C++"`?

I would also avoid an `_t` name, since that makes it look more like this is a name exposed by the library. Maybe even avoiding a real identifier entirely, and/or using an alternative typographical convention, would help to emphasize that this is not an exposed name:

> `using `*`atexit-handler`*` = void();              // `*`exposition only`*
> `extern "C" using `*`c-atexit-handler`*` = void(); // `*`exposition only`*
> `int atexit(`*`atexit-handler`*`* func);`
> `int atexit(`*`c-atexit-handler`*`* func);`
> `int at_quick_exit(`*`atexit-handler`*`* func);`
> `int at_quick_exit(`*`c-atexit-handler`*`* func);`

... although I note that none of our existing exposition-only names use such a typographical convention.